packages/api-explorer/src/components/Header/Header.tsx

```diff
@@ -53,8 +70,12 @@ export const Header: FC<HeaderProps> = ({ specs, spec, specDispatch }) => (
     pl="large"
   >
     <FlexItem>
-      <Heading color="palette.charcoal500" fontSize="xxlarge">
-        <NavLink to={`/${spec.key}`}>API Explorer</NavLink>
+      <Heading fontSize="xxlarge">
+        <Flex>
+          <StyledNavLink to={`/${spec.key}`}>
+            <Logo /> API Explorer
+          </StyledNavLink>
+        </Flex>
```

Comment:
Not 100% sure whats going on here, but do you need the `Flex` to wrap the `StyledNavLink`?  And I am not sure you need the `StyledNavLink` here either

I'd be curious to see what happens if you tried something like

```
<Heading fontSize="xxlarge">
  <NavLink to={`/${spec.key}`}>
   <Space gap="small">
      <Logo />
      <Text color="key"> API Explorer</Text>
   </Space>
  </NavLink>
</Heading>

Reply:
Cleaned this up a bit and used Space.